.env.example

```diff
@@ -10,5 +11,5 @@ MCP_SERVER_COMMAND=node build/index.mjs
 
 # Evaluation Configuration 
 # Note: This will run against all specified models for each provider
-EVAL_MODELS={"openai":"gpt-4o","anthropic":["claude-3-5-haiku-latest","claude-3-7-sonnet-latest"]}
+EVAL_MODELS={"openai":"gpt-4o","anthropic":["claude-3-5-haiku-latest","claude-3-7-sonnet-latest"],"gemini":"gemini-2.0-flash-001"}
```

Comment:
At some point we're gonna need to reckon with the fact that people won't use these models themselves in practice, but I don't think we wanna use the chonksters on every successful test run

Reply:
for PRs at least, we only run 4o-mini and 3.5-haiku.

might be worth changing the main evals to run on release rather than on pr merge tho?